parser/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -83,63 +87,67 @@ object LarCsvParser {
         )
       val denial = Denial(denial1, denial2, denial3)
 
-      Right(LoanApplicationRegister(
-        id,
-        respId,
-        agencyCode,
-        loan,
-        preapprovals,
-        actionType,
-        actionDate,
-        geography,
-        applicant,
-        purchaserType,
-        denial,
-        rateSpread,
-        hoepaStatus,
-        lienStatus
-      ))
+      Right(
+        LoanApplicationRegister(
+          id,
+          respId,
+          agencyCode,
+          loan,
+          preapprovals,
+          actionType,
+          actionDate,
+          geography,
+          applicant,
+          purchaserType,
+          denial,
+          rateSpread,
+          hoepaStatus,
+          lienStatus
+        )
+      )
     } else {
-      Left(larErrors)
+      val lErrors = validation.toEither.left.get
+      Left(lErrors.list.toList)
```

Comment:
This seems like a lot of conversion (Validation -> Either -> Value -> List -> back to Either).

There might be a simpler way of doing it, but I'm not sure what it could be